vim/syntax/replesent.vim

```diff
@@ -0,0 +1,7 @@
+" Vim syntax file
+" Language: REPLesent
+" Latest Revision: 2016-05-17
+" Depends on: https://github.com/derekwyatt/vim-scala
+
+syntax include @Scala syntax/scala.vim
+syntax region replesentBlock start=/```/ end=/```/ contains=@Scala
```

Comment:
I'm sure I can -- I'll be using REPLesent for my next presentation as well, where I'll probably want support for bold/colors -- having slide separators called out seems like a natural fit with that, and can definitely look into it.